src/token/erc721/erc721.cairo

```diff
@@ -195,34 +231,48 @@ mod ERC721 {
 
     #[external(v0)]
     impl ERC721CamelOnlyImpl of interface::IERC721CamelOnly<ContractState> {
+        /// Camel case support.
+        /// See [balance_of](balance_of).
```

Comment:
This is why I think it makes sense to add just one comment on top of the impl block, to avoid repeating these comments that don't add much.

For consistency we should add the

`/// Adds camelCase support for...`

on the impl, instead of each method.